app/controllers/clients_controller.rb

```diff
@@ -12,20 +12,31 @@ def show
     @movements = @client.movements
   end
 
+  # URL to call this would look like /clients/:client_id/eligibility/:date.json
+  # e.g. /clients/1/eligibility/2019-01-01.json
   def eligibility
     respond_to :json
 
-    @service = EligibilityService.new(Client.find(params[:id]), params[:date] || Date.now)
+    @service = EligibilityService.new(@client, requested_date)
     @service.run!
 
-    # URL to call this would look like /clients/eligibility?id=1&date="2019-01-01"
-    @service.to_json
+    render json: {
+      @requested_date => {
+        'meetsMinimumPresence' => @service.meets_minimum_presence_requirements[requested_date],
+        'last5Years' => @service.enough_days_by_rolling_year.values,
```

Comment:
This entry worries me. it's just an array of booleans - so there's no longer an indication that the first item in the array is the oldest year etc.

Reply:
Yeah, I noticed that as well. Is it documented in the service?

Reply:
The service wants to return like this:
```'2015-06-01' => 365, '2016-06-01' => 366, '2017-06-01' => 365, '2018-06-01' => 365, '2019-06-01' => 365```

but Mischa asked for it to change to 
```
[365, 366, 365, 365, 365]
```
because that's what the reactjs component is expecting.

Reply:
Yeah I was going back and forth on that, I liked it for simplicity but I think you are right that I've over-simplified it

Reply:
Can we add another array that is just the year numbers in the same order?
E.g.:
```
['2015', '2016', '2017', '2018', '2019']
```